test/distributed/_composable/fsdp/test_fully_shard_compile.py

```diff
@@ -63,7 +77,28 @@ def patched_trace_rules_check(*args, **kwargs):
             self.assertTrue(trace_rules_check_count > 0)
 
 
+def cleanup_fsdp(model):
```

Comment:
Is it insufficient to just del the module(s)? I would hope the owned tensors get garbage collected then without having to manually `resize_(0)`.

Reply:
A few weeks ago when I debug the memory usage issue, it isn't quite the case - `gc.collect` seems not able to free the owned tensors after the module is `del`-ed.

Alternatively we can remove this part and I can visit it again when I get to landing the memory tests for Inductor.

Reply:
Removed this function and will visit again in the future